Subject: [PATCH] Orion: use better key codes for the TS-209/TS-409 buttons

Use key codes for the buttons on the TS-209/TS-409 that make more sense
than the current values.

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>


--- a/arch/arm/mach-orion5x/ts209-setup.c	2008-07-23 14:13:08.000000000 +0000
+++ b/arch/arm/mach-orion5x/ts209-setup.c	2008-07-23 14:14:01.000000000 +0000
@@ -203,13 +203,13 @@
 
 static struct gpio_keys_button qnap_ts209_buttons[] = {
 	{
-		.code		= KEY_RESTART,
+		.code		= KEY_COPY,
 		.gpio		= QNAP_TS209_GPIO_KEY_MEDIA,
 		.desc		= "USB Copy Button",
 		.active_low	= 1,
 	},
 	{
-		.code		= KEY_POWER,
+		.code		= KEY_RESTART,
 		.gpio		= QNAP_TS209_GPIO_KEY_RESET,
 		.desc		= "Reset Button",
 		.active_low	= 1,
--- a/arch/arm/mach-orion5x/ts409-setup.c	2008-07-23 14:13:21.000000000 +0000
+++ b/arch/arm/mach-orion5x/ts409-setup.c	2008-07-23 14:14:13.000000000 +0000
@@ -212,7 +212,7 @@
 
 static struct gpio_keys_button qnap_ts409_buttons[] = {
 	{
-		.code		= KEY_RESTART,
+		.code		= KEY_COPY,
 		.gpio		= QNAP_TS409_GPIO_KEY_MEDIA,
 		.desc		= "USB Copy Button",
 		.active_low	= 1,

